config/sidekiq.yml

```diff
@@ -70,6 +70,10 @@
     args: ['send_notifications']
     description: "Notifies publishers their wallet is disconnected on the 23rd of every month."
     queue: scheduler
+  Sync::LaunchTokenRefreshJobsJob:
```

Comment:
I'm not a fan of this since this seems to generate a lot of unnecessary requests. We had something similar to this in the past and I believe we got rid of it.

Reply:
Removed, we'll just refresh on login and update the UI